docs/ops/convolution/Convolution_1.md

```diff
@@ -1,35 +1,35 @@
-## Convolution<a name="Convolution"></a> {#openvino_docs_ops_convolution_Convolution_1}
+## Convolution <a name="Convolution"></a> {#openvino_docs_ops_convolution_Convolution_1}
 
 **Versioned name**: *Convolution-1*
 
 **Category**: Convolution
 
-**Short description**: [Reference](http://caffe.berkeleyvision.org/tutorial/layers/convolution.html)
+**Short description**: Computes 1D, 2D or 3D convolution (cross-correlation to be precise) of input and kernel tensors.
 
-**Detailed description**: [Reference](http://cs231n.github.io/convolutional-networks/#conv)
+**Detailed description**: [Convolutional Neural Networks](http://cs231n.github.io/convolutional-networks/#conv), [Convolution operation](https://medium.com/apache-mxnet/convolutions-explained-with-ms-excel-465d6649831c).  
 
-
-*   For the convolutional layer, the number of output features in each dimension is calculated using the formula:
+For the convolutional layer, the number of output features in each dimension is calculated using the formula:  
 \f[
 n_{out} = \left ( \frac{n_{in} + 2p - k}{s} \right ) + 1
-\f]
-*   The receptive field in each layer is calculated using the formulas:
-    *   Jump in the output feature map:
-        \f[
-        j_{out} = j_{in} * s
-        \f]
-    *   Size of the receptive field of output feature:
-        \f[
-        r_{out} = r_{in} + ( k - 1 ) * j_{in}
-        \f]
-    *   Center position of the receptive field of the first output feature:
-        \f[
-        start_{out} = start_{in} + ( \frac{k - 1}{2} - p ) * j_{in}
-        \f]
-    *   Output is calculated using the following formula:
-        \f[
-        out = \sum_{i = 0}^{n}w_{i}x_{i} + b
-        \f]
+\f] 
+
+The receptive field in each layer is calculated using the formulas:  
+*   Jump in the output feature map:  
+  \f[
+  j_{out} = j_{in} * s
+  \f]
+*   Size of the receptive field of output feature:  
+  \f[
+  r_{out} = r_{in} + ( k - 1 ) * j_{in}
+  \f]
+*   Center position of the receptive field of the first output feature:  
+  \f[
+  start_{out} = start_{in} + ( \frac{k - 1}{2} - p ) * j_{in}
+  \f]
+*   Output is calculated using the following formula: 
+  \f[
+  out = \sum_{i = 0}^{n}w_{i}x_{i} + b
+  \f]
 
 **Attributes**
```

Comment:
Please, wrap strings like "(z, y, x)", with a back-tick like this "\`(z, y, x)\`". Also, please, replace "*4,2,1*" with code like this "\`4, 2, 1\`".